### PR TITLE
Hello Microverse Project

### PR DESCRIPTION
**Pull request by Bhekisile**
Add linters and gitignore files
Add html text
Add style to html

### CODE REVIEW

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @Bhekisile, 

Welcome to Microverse🤝

# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

A little bit more

## Good Highlights✔️

- Good commit messages
- Descriptive pull request


## Required Changes ♻️

1. Every project is expected to be properly documented with details including how other programmers can run the project on their machines. As stated in the [project requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/083a7b724f96d81883401e8d6602aeac363301d4/documentation/hello_microverse_project.md?plain=1#L40), kindly use [this template](https://github.com/microverseinc/readme-template) by Microverse to include a descriptive readme for your project, customizing the sections to match your current project.
2. You have made a good attempt at installing linters, but there's more. Using [this guide](https://github.com/microverseinc/linters-config/tree/master/html-css) for setting up linters for a HTML/CSS project, you'll see that `linters.yml` file is supposed to be in `workflows` folder and workflows folder should be in `.github` folder. You may also follow [this part of the instructions](https://github.com/microverseinc/linters-config/tree/master/html-css#set-up-linters-in-your-local-env) to set up linters in your local environment so that you can find and fix errors before pushing to GitHub.
3. The .github folder conventionally contains the github actions setup for the project. Also, most hosting platforms expect the entry point (index.html) of the website to be located in the root folder of the project. So, kindly move the `index.html` and `styles.css` files to the root folder of the project.

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1 +1,55 @@
...
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Bhekisile Jozi**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
```
Comment (franclobo) [inline README.md]: - [ ] As the last code reviewer said, all main headers from the [README template](https://github.com/microverseinc/readme-template) should be present in your repo, please 🙏🏽 complete your README file with all sections: 👇🏽 

> Every project is expected to be properly documented with details including how other programmers can run the project on their machines. As stated in the [project requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/083a7b724f96d81883401e8d6602aeac363301d4/documentation/hello_microverse_project.md?plain=1#L40), kindly use [this template](https://github.com/microverseinc/readme-template) by Microverse to include a descriptive readme for your project, customizing the sections to match your current project.

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @Bhekisile, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Correct Github flow ✔️ 
- Good implementation of suggestions from code reviewer ✔️ 


 ## Required Changes ♻️
_Check the comments under the review._

- [ ] As the last code reviewer said, please follow the instructions to install linters, the linters.yml file should be inside this folders `.github/workflows/`: 👇🏽 

> You have made a good attempt at installing linters, but there's more. Using [this guide](https://github.com/microverseinc/linters-config/tree/master/html-css) for setting up linters for a HTML/CSS project, you'll see that linters.yml file is supposed to be in workflows folder and workflows folder should be in .github folder. You may also follow [this part of the instructions](https://github.com/microverseinc/linters-config/tree/master/html-css#set-up-linters-in-your-local-env) to set up linters in your local environment so that you can find and fix errors before pushing to GitHub.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by franclobo — APPROVED** [review]
Hi @Bhekisile, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Descriptive PR ✔️ 
- Professional README file ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**